portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,69 @@
+#background {
+    
+}
+
+#bar1,
+#bar2,
+#bar3 {
+  width: 35px;
+  height: 5px;
+  background-color: black;
```

Comment:
Prefer the [`background` shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/background) to `background-color`, per our style guide:
https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties
(here and below at line 37 below).